### PR TITLE
Add HaGeZi Personal Black & White

### PR DESCRIPTION
**Pull request by hagezi**
@ameshkov as discussed: https://github.com/AdguardTeam/HostlistsRegistry/pull/101

### CODE REVIEW

File: filters/general/filter_34_HageziPersonal/metadata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "filterId": "hagezi_personal",
+  "id": 34,
+  "name": "HaGeZi Personal Black & White",
+  "description": "An extension for existing blocklists, my personal black and whitelist. Unblocks false positives and referral domains.",
+  "timeAdded": 1667987017479,
+  "homepage": "https://github.com/hagezi/dns-blocklists",
+  "expires": "4 days",
+  "displayNumber": 4,
+  "environment": "prod",
```
Comment (ameshkov) [inline filters/general/filter_34_HageziPersonal/metadata.json:10]: Could you please set it to `dev` for the time being? We'll update it to prod when it's tested privately

**Review by ameshkov — CHANGES_REQUESTED** [review]
Looks great, thank you!

Just one little thing to do, we need it to go through internal testing first.
Could you please set the environment to `dev` for now?

File: filters/general/filter_34_HageziPersonal/metadata.json (hunk 1 above)
Comment (hagezi) [reply filters/general/filter_34_HageziPersonal/metadata.json]: Updated

**Comment by ameshkov** [comment]
Awesome, thank you!

**Comment by hagezi** [comment]
@ameshkov Thanks for your support!

**Comment by marcoclg19** [comment]
When will this blocklist be available? @ameshkov

**Comment by ameshkov** [comment]
Next week alongside the v2.1.3 update.

**Pull request merged**